tools/monitoring/gke_deployment/monitoring_service.yaml

```diff
@@ -41,11 +41,23 @@ spec:
       containers:
       - name: monitoring-app
         image: gcr.io/knative-tests/test-infra/monitoring:latest
+        command: ["/monitoring"]
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
can we use the secret from the mount without using this env var?

Reply:
This is for authenticating to the cloud api. From the code we write, we actually don't do anything with this secret / environment variable, but the cloud mail code fails authentication without this. 
https://cloud.google.com/docs/authentication/getting-started#setting_the_environment_variable